src/components/SearchBar.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Input, Button } from '@material-ui/core';
+import axios from 'axios';
+
+const API_KEY = 'AIzaSyDAfW_5XoEHmW6r0KsxFo9xJ6DYSuk5RH8';
```

Comment:
I haven't never used youtube api and don't know what it can do, but I feel unsafe to write a api key in a public repo.
In such cases, I have used the following package and hidden a api key.
https://www.npmjs.com/package/dotenv